blank-canvas-blocks/sass/blocks/_gallery.scss

```diff
@@ -0,0 +1,21 @@
+.wp-block-gallery {
+
+	margin: var(--wp--custom--gallery--margin);
+
+	.blocks-gallery-image,
+	.blocks-gallery-item {
+
+		figcaption {
+			margin: var(--wp--custom--gallery--caption--margin);
+			// Text color is always white to account for default gradient background
+			color: var(--wp--custom--gallery--caption--color);
+			font: var(--wp--custom--gallery--caption--font-size);
```

Comment:
Should this be `font-size` or are you intentionally using the shorthand? If so, I believe font family also needs to be supplied in the value.